changelog/unreleased/issue-340

```diff
@@ -0,0 +1,8 @@
+Enhancement: Add support for Volume Shadow Copy Service (VSS) on windows
```

Comment:
s/windows/Windows/

Reply:
Changed in https://github.com/restic/restic/pull/2274/commits/0d6ae7c3b64e39fc9872184a775b31f814440ab5